lib/mandrill-transport.js

```diff
@@ -54,7 +60,18 @@ MandrillTransport.prototype.send = function(mail, callback) {
   if (data.preserve_recipients !== undefined) {
     preserve_recipients = data.preserve_recipients;
   }
-
+  if (data.attachments !== undefined) {
+    // merge sent attachments with previously defined common default attachments
+    for (var i in data.attachments) {
+      var attachment = data.attachments[i];
+      attachments.push({
+        'name': attachment.filename,
+        'type': attachment.contentType,
+        'content': attachment.content.toString('base64')
```

Comment:
This assumes that content is of type `Buffer`
But it can be a string too. `toString('base64')` does not do the right thing when called on a string.